spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -65,20 +70,22 @@ public String send(final String topic, Message message) throws RuntimeException
 			try {
 				return Publisher.defaultBuilder(TopicName.create(projectId, topic))
```

Comment:
Regarding message conversion chaining, I'm not so sure how that would work. All the examples I've seen use a single converter, so I suggest to just add a setter/getter in this PR and add chaining later?